src/Umbraco.Web.UI.Client/src/packages/documents/documents/publishing/schedule-publish/modal/document-schedule-modal.element.ts

```diff
@@ -206,34 +215,68 @@ export class UmbDocumentScheduleModalElement extends UmbModalBaseElement<
 				<div>
 					<umb-input-date
 						type="datetime-local"
+						.value=${this.#formatDate(option.variant?.scheduledUnpublishDate)}
 						@change=${(e: Event) => this.#onToDateChange(e, option.unique)}
 						label=${this.localize.term('general_publishDate')}></umb-input-date>
 				</div>
 			</uui-form-layout-item>
 		</div>`;
 	}
 
+	/**
+	 * Formats the date to be compatible with the input type datetime-local
+	 * @param {string} dateStr The date to format, example: 2021-01-01T12:00:00.000+01:00
+	 * @returns {string | undefined} The formatted date in local time with no offset, example: 2021-01-01T11:00
+	 */
+	#formatDate(dateStr?: string | null): string | undefined {
+		if (!dateStr) return undefined;
+
+		const d = new Date(dateStr);
+
+		if (isNaN(d.getTime())) {
+			console.warn('[Schedule]: Invalid date:', dateStr);
+			return undefined;
+		}
+
+		// We need to subtract the offset to get the correct time in the input field
+		// the input field expects local time without offset and the Date object will convert the date to local time
+		return (
+			d.getFullYear() +
+			'-' +
+			String(d.getMonth() + 1).padStart(2, '0') +
+			'-' +
+			String(d.getDate()).padStart(2, '0') +
+			'T' +
+			String(d.getHours()).padStart(2, '0') +
+			':' +
+			String(d.getMinutes()).padStart(2, '0')
+		);
```

Comment:
I'm wondering if a substring is simpler to remove the seconds/milliseconds part?

```suggestion
		return d.toISOString().slice(0, -8);
```

Reply:
Maybe but are we sure that function returns the same format across all OS'es and browsers?

Reply:
[MDN suggests that it should, (based on ISO 8601)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString), but then mentions an alternative `±YYYYYY-MM-DDTHH:mm:ss.sssZ` format, (with a six digit year! 🤯), which has thrown me into doubt, so I can't say for sure.

Let's leave the code as it is. 👍